File: reverse-word/ReverseWord.js
function reverseWord(str) {
    let words = str.split(" ");
    for(let i = 0; i < words.length / 2; i++) {
        let lastIndex = words.length - i - 1;
        let temp = words[i];
        words[i] = words[lastIndex];
        words[lastIndex] = temp;
    }
    return words.join(" ");
}

exports.reverseWord = reverseWord;

File: reverse-word/ReverseWord.py
class Solution:
    def ReverseWords(self, s: str) -> str:
        words = s.split(" ")
        words = [word for word in words if word != '']
        for index in range(0, len(words) // 2):
            last_index = len(words) - 1 - index
            temp = words[index]
            words[index] = words[last_index]
            words[last_index] = temp
        return " ".join(words).strip()